### PR TITLE
Add issue templated for docs and update existing issue templates

### LINKED ISSUE #3205: Redefine issue templates
Currently there is only the following issue templates.

- Icon request
- Icon update
- Packages
- Website

If someone asks a question that doesn't fit in the a valid category they will most likely choose an incorrect one.

There is no template for documentation, or general\ misc ideas.
More issue types should be included to avoid blank issues or the incorrect type being selected.

### PR DESCRIPTION
**Pull request by ericcornelissen**
**Issue:** Closes #3205 (?)
**Alexa rank:** n/a
  

### Checklist
  - [n/a] I updated the JSON data in `_data/simple-icons.json`
  - [n/a] I optimized the icon with SVGO or SVGOMG
  - [n/a] The SVG `viewbox` is `0 0 24 24`

### Description

1. As per #3205, add an issue template for issues regarding documentation
2. Improve the descriptions of existing issue templates

I was also thinking about adding an issue template named something like "Suggestions" that may nudge people to create an issue instead of a discussion like https://github.com/simple-icons/simple-icons/discussions/5614, https://github.com/simple-icons/simple-icons/discussions/5530, and https://github.com/simple-icons/simple-icons/discussions/5186 but I felt like that should already be covered by the [package issue template](https://github.com/simple-icons/simple-icons/blob/9e7f8c97914fef2678a569d356b659016987b29e/.github/ISSUE_TEMPLATE/package.md). Hence, I decided to update the description instead. If anyone thinks it could still be useful to have a "Suggestions" issue template, leave a comment with your suggestion (e.g. an example `.md` template and/or suggested labels).

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
LGTM, @ericcornelissen! Will wait on feedback from other @simple-icons/maintainers before merging though.

**Review by service-paradis — APPROVED** [review]
LGTM :)

**Review by mondeja — APPROVED** [review]

**Pull request merged**
